docs/schema/accountauthcredentials-complextype-sps15xsdsearchset2.md

```diff
@@ -1,24 +1,24 @@
 ---
 title: "AccountAuthCredentials complexType (SPS15XSDSearchSet2)"
+description: Discusses the definition, elements and attributes for AccountAuthCredentials complexType (SPS15XSDSearchSet2).
 manager: arnek
-ms.date: 3/9/2015
+ms.date: 06/13/2022
 ms.audience: ITPro
 ms.topic: article
 ms.prod: sharepoint
 ms.localizationpriority: medium
 ms.assetid: fb7d56a6-5985-2379-7183-6bee6fcd3f90
-description: "Last modified: March 09, 2015"
 ---
 
 # AccountAuthCredentials complexType (SPS15XSDSearchSet2)
 
 ## Type information
 
-|||
+|Information|Location|
 |:-----|:-----|
-|**Namespace** <br/> |http://schemas.datacontract.org/2004/07/Microsoft.Office.Server.Search.Administration  <br/> |
-|**Schema file** <br/> |schema_Microsoft.Office.Server.Search.Administration.xsd  <br/> |
-|**Extension base** <br/> |tns:AuthenticationData  <br/> |
+|**Namespace** </br> |http://schemas.datacontract.org/2004/07/Microsoft.Office.Server.Search.Administration  </br> |
```

Comment:
When you see these HTML breaks that don't do anything, feel free to delete the `<br/>`.